diff -ur vde2-2.2.2.orig/src/kvde_switch/consmgmt.c vde2-2.2.2/src/kvde_switch/consmgmt.c
--- vde2-2.2.2.orig/src/kvde_switch/consmgmt.c	2008-06-15 18:49:19.000000000 +0200
+++ vde2-2.2.2/src/kvde_switch/consmgmt.c	2009-02-24 20:03:49.000000000 +0100
@@ -24,6 +24,7 @@
 #include <getopt.h>
 #include <dlfcn.h>
 #include <time.h>
+#include <limits.h>
 
 #include <config.h>
 #include <vde.h>
diff -ur vde2-2.2.2.orig/src/kvde_switch/datasock.c vde2-2.2.2/src/kvde_switch/datasock.c
--- vde2-2.2.2.orig/src/kvde_switch/datasock.c	2008-06-15 18:49:19.000000000 +0200
+++ vde2-2.2.2/src/kvde_switch/datasock.c	2009-02-24 20:04:38.000000000 +0100
@@ -26,6 +26,7 @@
 #include <grp.h>
 #define _GNU_SOURCE
 #include <getopt.h>
+#include <limits.h>
 
 #include <config.h>
 #include <vde.h>
Only in vde2-2.2.2/src/kvde_switch: datasock.c~
diff -ur vde2-2.2.2.orig/src/vde_plug2tap.c vde2-2.2.2/src/vde_plug2tap.c
--- vde2-2.2.2.orig/src/vde_plug2tap.c	2008-03-10 10:10:46.000000000 +0100
+++ vde2-2.2.2/src/vde_plug2tap.c	2009-02-24 20:09:10.000000000 +0100
@@ -18,6 +18,7 @@
 #include <libgen.h>
 #define _GNU_SOURCE
 #include <getopt.h>
+#include <limits.h>
 
 #include <config.h>
 #include <vde.h>
Only in vde2-2.2.2/src: vde_plug2tap.c~
diff -ur vde2-2.2.2.orig/src/wirefilter.c vde2-2.2.2/src/wirefilter.c
--- vde2-2.2.2.orig/src/wirefilter.c	2008-07-07 07:56:22.000000000 +0200
+++ vde2-2.2.2/src/wirefilter.c	2009-02-24 20:13:53.000000000 +0100
@@ -31,6 +31,7 @@
 #include <sys/stat.h>
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <limits.h>
 
 #include <config.h>
 #include <vde.h>
Only in vde2-2.2.2/src: wirefilter.c~
