--- vde2-2.2.2/src/lib/Makefile.am.orig	2009-02-23 22:59:20.000000000 +0100
+++ vde2-2.2.2/src/lib/Makefile.am	2009-02-23 23:07:35.000000000 +0100
@@ -11,7 +11,7 @@
 	libvdemgmt.la \
 	libvdeplug.la
 
-libvdesnmp_la_LIBADD = $(LIBADD)
+libvdesnmp_la_LIBADD = $(LIBADD) libvdemgmt.la
 libvdesnmp_la_LDFLAGS = $(AM_LDFLAGS) -version-number 0:0:1 -export-dynamic
 
 libvdemgmt_la_LIBADD = $(LIBADD)
--- vde2-2.2.2/src/lib/Makefile.in~	2008-07-08 10:38:15.000000000 -0400
+++ vde2-2.2.2/src/lib/Makefile.in	2009-10-03 13:00:41.000000000 -0400
@@ -62,7 +62,7 @@
 libvdeplug_la_LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) \
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(libvdeplug_la_LDFLAGS) $(LDFLAGS) -o $@
-libvdesnmp_la_DEPENDENCIES = $(LIBADD)
+libvdesnmp_la_DEPENDENCIES = $(LIBADD) ./libvdemgmt.la
 libvdesnmp_la_SOURCES = libvdesnmp.c
 libvdesnmp_la_OBJECTS = libvdesnmp.lo
 libvdesnmp_la_LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) \
@@ -202,7 +202,7 @@
 	libvdemgmt.la \
 	libvdeplug.la
 
-libvdesnmp_la_LIBADD = $(LIBADD)
+libvdesnmp_la_LIBADD = $(LIBADD) ./libvdemgmt.la
 libvdesnmp_la_LDFLAGS = $(AM_LDFLAGS) -version-number 0:0:1 -export-dynamic
 libvdemgmt_la_LIBADD = $(LIBADD)
 libvdemgmt_la_LDFLAGS = $(AM_LDFLAGS) -version-number 0:0:1 -export-dynamic
